packages/labs/test/tagInputTests.tsx

```diff
@@ -343,6 +343,22 @@ describe("<TagInput>", () => {
         );
     });
 
+    it("is non-interactive when disabled", () => {
+        const wrapper = mount(<TagInput values={VALUES} disabled={true} />);
+
+        assert.isTrue(wrapper.hasClass(Classes.DISABLED), `.${Classes.DISABLED} should be applied to .pt-tag-input`);
+        assert.isTrue(
+            wrapper
+                .find(".pt-input-ghost")
+                .first()
+                .prop("disabled"),
+            ".pt-input-ghost does not have a 'disabled' attribute",
```

Comment:
nit: Use the same language here to clarify whether this is what the state should be vs what it is: `.pt-input-ghost should have a 'disabled' attribute`